docs/OV_Runtime_UG/hetero_execution.md

```diff
@@ -103,24 +115,24 @@ If you want different devices in Hetero execution to have different device-speci
 
 @endsphinxdirective
 
-In the example above, `GPU` device is configured to enable profiling data, while only `CPU` device has configuration property to perform inference in `f32` precision, while GPU has default execution precision.
+In the example above, the `CPU` device is configured to enable profiling data. `GPU` has the configuration property to perform inference in `f16` precision, while CPU has the default execution precision.
```

Comment:
looks like you mixed the devices. We need:
- profiling for GPU
- fp32 for CPU

Reply:
I have no idea how and why the discrepancy :) Thanks, corrected accordingly.